src/CMakeLists.txt

```diff
@@ -1006,6 +1030,7 @@ set (PCH_SOURCES
 	gl/stereo3d/gl_stereo_leftright.cpp
 	gl/stereo3d/scoped_view_shifter.cpp
 	gl/stereo3d/gl_anaglyph.cpp
+    gl/stereo3d/gl_openvr.cpp
```

Comment:
Please use tabs. There are few cases of commits with spaces (including mine, I'm sorry for that) and they are causing only problems.